### PR TITLE
Add a Standalone Cell Example

### PR DESCRIPTION
**Pull request by blink1073**
cc @minrk 

- Starts the default kernel and includes a toolbar that allows for switching kernels.
- Includes syntax highlighting, completion, and tooltips
- Also includes some housekeeping for the other examples

image

### CODE REVIEW

File: examples/cell/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  let panel = new BoxPanel();
+  panel.id = 'main';
+  panel.direction = 'top-to-bottom';
+  panel.spacing = 0;
+  panel.addWidget(completer);
+  panel.addWidget(toolbar);
+  panel.addWidget(cellWidget);
+  BoxPanel.setStretch(toolbar, 0);
+  BoxPanel.setStretch(cellWidget, 1);
+  Widget.attach(panel, document.body);
```
Comment (sccolbert) [inline examples/cell/src/index.ts]: I would separate this `attach` call with a blank line and a comment. It is easy to miss otherwise.

File: packages/apputils/src/clientsession.ts [hunk 2]
```diff
@@ -211,6 +211,11 @@ namespace IClientSession {
      * Whether a kernel can be started.
```
Comment (sccolbert) [inline packages/apputils/src/clientsession.ts]: Should changes to this file be in a different PR?

File: examples/cell/src/index.ts (hunk 1 above)
Comment (minrk) [reply examples/cell/src/index.ts]: That would be helpful. One thing I had trouble with when trying to apply these examples outside was how much of this was necessary vs incidental. e.g. I'm not using any panels, but since I have a single Widget, I still need to call `.attach`, etc.

Comment (blink1073) [reply examples/cell/src/index.ts]: Yep, done in #3158
